lom/src/plugins/plugins_common/plugins_common.go

```diff
@@ -178,7 +180,7 @@ func (gpl *PluginMetadata) CheckMisbehavingPlugins(pluginKey string) bool {
     gpl.response[pluginKey] = responses
 
     // Check if the window size has reached the limit
-    if len(responses) >= MAX_PLUGIN_RESPONSES_DEFAULT {
+    if len(responses) >= gpl.MaxPluginResponses {
```

Comment:
This can precede updating responses. Why do update & then delete. Skip update, if to be deleted

if len(responses) - skip + 1 > ... {
    delete 
    return
}
// update here

Reply:
TODO